values.html

```diff
@@ -265,27 +235,27 @@ <h2 class="h2-styles">WE SUPPORT</h2>
 					/>
 				</svg>
 			</div>
-			<div class="support-text">
+			<div class="support-text wrapper">
```

Comment:
The heading text in the UN section is too wide. It needs a max-width on desktop so that it drops to the second line sooner. 

<img width="1404" alt="Screen Shot 2024-01-30 at 10 32 52 AM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/4526c126-c989-47b2-b283-7e1abf786940">

Reply:
Added max-width to .support p so the text goes to the second line for desktop